gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -0,0 +1,654 @@
++
+----
+Error: UPGRADE FAILED: execution error at (flex-gateway/templates/deployment.yaml:4:10):
+registerSecretName, registration.content or registration.secretName is required!
+----
++
+When this error occurs, check the `registration.yaml`. Make sure the file is not empty. If the file is empty, re-register your gateway with a new, unique name to generate a new `registration.yaml` (see <<register-flex>>) and rerun the procedures that follow the registration step. 
+
+[[cluster_unreachable]]
+* If your Kubernetes cluster is not running when you attempt to deploy Flex Gateway using `helm`, the following error can occur:
++
+----
+Error: Kubernetes cluster unreachable: 
+the server could not find the requested resource
+----
++
+Start the service that runs your cluster. 
++
+You can check that your Kubernetes cluster host is running your cluster, for example:
++
+[source,kubectl]
+----
+kubectl cluster-info
+----
++
+If the cluster is running, the command returns a result similar to this one:
++
+----
+Kubernetes control plane is running at https://34.30.50.119
+GLBCDefaultBackend is running at https://34.30.50.119/api/v1/namespaces/kube-system/services/default-http-backend:http/proxy
+KubeDNS is running at https://34.30.50.119/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+Metrics-server is running at https://34.30.50.119/api/v1/namespaces/kube-system/services/https:metrics-server:/proxy
+----
++
+If the cluster is not up, the command can return a result similar to this one:
++
+----
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+error: the server doesn't have a resource type "services"
+----
++
+For additional information, see <<check-cluster>>.
+
+[[upgrade-failed]]
+* Some issues can cause the Helm upgrade command to hang for up to 5 minutes (due to the `--wait` flag in the command) before producing the following timeout error:
++
+----
+Error: UPGRADE FAILED: timed out waiting for the condition
+----
++
+This error typically means that your cluster configuration is incorrect or that your cluster configuration tool does not support this process. See <<k8-cluster-reqs>>. One potential cause is the failure to generate a `Service` resource of the `LoadBalancer` type. You can troubleshoot further by running the following commands:
+
+** {empty}
++
+[source,kubectl]
+----
+kubectl -n gateway describe deployment
+----
+
+** {empty}
++
+[source,kubectl]
+----
+kubectl -n gateway describe service
+----
+
+//TODO: this was from Israel. Not clear what upgrade error and 
+//      why we would uninstall the ingress for the namespace. 
+//Israel: "If you receive an upgrade error when execute the command, you can  
+//         remove the ingress released chart or omit the released argument:"
+//
+//## Command to remove ingress chart released
+//helm -n <namespace> uninstall ingress
```

Comment:
same as helm delete my-release, which is already doc'd https://docs.mulesoft.com/gateway/1.4/flex-gateway-uninstall#uninstall-as-a-kubernetes-ingress-controller